openeo-geotrellis/src/main/scala/org/openeo/geotrellis/layers/NetCDFCollection.scala

```diff
@@ -28,99 +27,130 @@ object NetCDFCollection {
   def datacube_seq(polygons:ProjectedPolygons, from_date: String, to_date: String,
                    metadata_properties: util.Map[String, Any], correlationId: String, dataCubeParameters: DataCubeParameters,osClient:OpenSearchClient): Seq[(Int, MultibandTileLayerRDD[SpaceTimeKey])] = {
     val sc = SparkContext.getOrCreate()
-    val cube: RDD[(SpaceTimeKey, MultibandTile)] with Metadata[TileLayerMetadata[SpaceTimeKey]] = loadCollection(osClient, sc)
-    Seq((0, cube))
-  }
-
-  def loadCollection(osClient:OpenSearchClient,sc: SparkContext) = {
     val stacItems = osClient.getProducts("", None, null, Map[String, Any](), "", "")
-    val items = sc.parallelize(stacItems)
 
-    sc.setJobDescription(s"load_stac from netCDFs - ${stacItems.head.id} - ${stacItems.head.links.head.href}")
-    val resolutions = items.flatMap(_.resolution).distinct().collect()
-    if(resolutions.length != 1) {
-      throw new IllegalArgumentException("All items in a collection must have the same resolution")
-    }
+    val cube: RDD[(SpaceTimeKey, MultibandTile)] with Metadata[TileLayerMetadata[SpaceTimeKey]] =
+      if (stacItems.nonEmpty) {
+        loadCollection(stacItems, sc)
+      } else {
+        loadEmptyCollection(polygons, from_date, dataCubeParameters, sc)
+      }
+    Seq((0, cube))
+  }
 
-    val crs = items.flatMap(_.crs).distinct().collect()
-    if(crs.length != 1) {
-      throw new IllegalArgumentException("All items in a collection must have the same CRS")
-    }
+  private def loadEmptyCollection(polygons: ProjectedPolygons, from_date: String, dataCubeParameters: DataCubeParameters, sc: SparkContext) = {
+    val boundingBox = polygons.extent
 
-    val bboxWGS84: Extent = items.map(_.bbox).reduce((a, b)=>(a.combine(b)))
+    val from = ZonedDateTime.parse(from_date, ISO_OFFSET_DATE_TIME)
+    val to = ZonedDateTime.parse("2025-06-01T00:00Z", ISO_OFFSET_DATE_TIME)
 
+    val scheme = FloatingLayoutScheme(dataCubeParameters.tileSize)
+    val maxSpatialResolution = CellSize(10, 10)
```

Comment:
Would it make sense to get resolution from the stac metadata from stac?
For the data type, I am not sure if it is even available from the stac metadata